examples/nginx-ldap-auth/config/my-ldap.config.yaml

```diff
@@ -0,0 +1,10 @@
+web: 0.0.0.0:5555
+path: /
+servers:
+  - ldap://ldap-server
+auth:
+  bindDN: cn=admin,dc=sighup,dc=io
+  bindPW: HatFriday
+user:
+  baseDN: ou=group-a,ou=system,dc=sighup,dc=io
+  filter: "(cn={0})"
```

Comment:
I think it would be nice to add some comments to the configuration file explaining what does each line/parameter, some are self-explanatory but others not so much.

Reply:
Adding it! Good idea.